### PR TITLE
add ng tests

### PR DESCRIPTION
**Pull request by vladimir-nechai**
ng test script added to travis CI

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -71,6 +71,10 @@ java_tests:
 fintech-ui/node_modules:
 	cd fintech-examples/fintech-ui && npm install
 
+.PHONY : fintech-ui-test
```
Comment (valb3r) [inline Makefile]: Probably we need to report coverage to codecov.io, not sure will it be picked automatically or not. But even if it does it is good idea to report coverage under separate label:
https://docs.codecov.io/docs/merging-reports#section-multiple-languages

File: .travis.yml [hunk 2]
```diff
@@ -12,6 +12,7 @@ jdk:
 script:
   # tests are not done with mvn test - dont know why
   - ./scripts/build_mvn.sh
+  - docker run -it --rm -v "$PWD":/opt/app-root/src -w /opt/app-root/src -u "$(id -u "${USER}"):$(id -g "${USER}")" trion/ng-cli-karma make fintech-ui-test
```
Comment (electronicpeter) [inline .travis.yml]: looks like travis is using other angular version than expected. Build fails due to this:

Makefile:76: recipe for target 'fintech-ui-test' failed
make: *** [fintech-ui-test] Error 1
travis_time:end:1cf331f5:start=1582282152590755213,finish=1582282212945483794,duration=60354728581,event=script
[0K[31;1mThe command "docker run -it --rm -v "$PWD":/opt/app-root/src -w /opt/app-root/src -u "$(id -u "${USER}"):$(id -g "${USER}")" trion/ng-cli-karma make fintech-ui-test" exited with 2.[0m

Comment (vladimir-nechai) [reply .travis.yml]: fix the failure

**Comment by vladimir-nechai** [comment]
was resolved

**Pull request closed without merging**